### PR TITLE
SAAS-1154 migrate ManageOrganization to MUI

### PR DESCRIPTION
**Pull request by nicolalamacchia**

### CODE REVIEW

File: src/pages/ManageOrganization/ManageOrganization.styles.ts [hunk 1]
```diff
@@ -1,47 +1,31 @@
...
+  container: css`
+    color: ${palette.text.primary};
+    display: flex;
+    flex-direction: column;
+    flex: 1;
 
-  return {
-    container: css`
-      color: ${colors.text};
+    h5 {
```
Comment (TomasBlazek3PG) [inline src/pages/ManageOrganization/ManageOrganization.styles.ts:11]: Please use a className

File: src/pages/ManageOrganization/ManageOrganization.styles.ts [hunk 2]
```diff
@@ -1,47 +1,31 @@
...
-      }
-    `,
-    disabledTab: css`
-      cursor: not-allowed;
+      margin: 0 0 ${spacing(2)};
 
-      > * {
-        color: ${colors.linkDisabled};
-        pointer-events: none;
+      svg {
```
Comment (TomasBlazek3PG) [inline src/pages/ManageOrganization/ManageOrganization.styles.ts:16]: the same

File: src/pages/ManageOrganization/OrganizationCreate/OrganizationCreate.styles.ts [hunk 3]
```diff
@@ -1,34 +1,32 @@
...
-    orgNameInput: css`
-      width: 200px;
-    `,
-    title: css`
-      font-size: ${typography.heading.h4};
-      margin: ${spacing.md} 0 ${spacing.lg};
-    `,
-  };
-};
+    svg {
```
Comment (TomasBlazek3PG) [inline src/pages/ManageOrganization/OrganizationCreate/OrganizationCreate.styles.ts:15]: Again please use classNames

File: src/pages/ManageOrganization/OrganizationEdit/OrganizationEdit.styles.ts [hunk 4]
```diff
@@ -1,43 +1,39 @@
...
+    padding-top: ${spacing(4)};
 
-  return {
-    container: css`
-      align-items: center;
-      display: flex;
-      flex-direction: column;
-      justify-content: center;
-      padding-top: ${spacing.lg};
+    svg {
```
Comment (TomasBlazek3PG) [inline src/pages/ManageOrganization/OrganizationEdit/OrganizationEdit.styles.ts:12]: classNames

File: src/pages/ManageOrganization/OrganizationView/OrganizationView.styles.ts [hunk 5]
```diff
@@ -1,61 +1,57 @@
...
+    padding-top: ${spacing(4)};
 
-  return {
-    container: css`
-      align-items: center;
-      display: flex;
-      flex-direction: column;
-      justify-content: center;
-      padding-top: ${spacing.lg};
+    & > svg {
```
Comment (TomasBlazek3PG) [inline src/pages/ManageOrganization/OrganizationView/OrganizationView.styles.ts:12]: classNames

**Review by TomasBlazek3PG — CHANGES_REQUESTED** [review]
Oh and I found one visual problem:
image

File: src/pages/ManageOrganization/ManageOrganization.styles.ts (hunk 2 above)
Comment (nicolalamacchia) [reply src/pages/ManageOrganization/ManageOrganization.styles.ts:16]: this is pointing to the internal svg, cannot be targeted by a class

File: src/pages/ManageOrganization/OrganizationEdit/OrganizationEdit.styles.ts (hunk 4 above)
Comment (nicolalamacchia) [reply src/pages/ManageOrganization/OrganizationEdit/OrganizationEdit.styles.ts:12]: see above

File: src/pages/ManageOrganization/OrganizationCreate/OrganizationCreate.styles.ts (hunk 3 above)
Comment (nicolalamacchia) [reply src/pages/ManageOrganization/OrganizationCreate/OrganizationCreate.styles.ts:15]: see above

File: src/pages/ManageOrganization/OrganizationView/OrganizationView.styles.ts (hunk 5 above)
Comment (nicolalamacchia) [reply src/pages/ManageOrganization/OrganizationView/OrganizationView.styles.ts:12]: see above

File: src/pages/ManageOrganization/InviteMember/InviteMember.tsx [hunk 6]
```diff
@@ -20,37 +25,41 @@ import { Messages } from './InviteMember.messages';
...
+
+  const validationSchema = yup.object({
+    invitedUsers: yup.array().of(
+      yup.object({
+        username: yup
+          .string()
+          .email()
+          .label(Messages.email)
+          .required()
+          .test('invalidUser', 'an error was found', (value) => {
```
[the remark this answers is not in the document]
Comment (nicolalamacchia) [reply src/pages/ManageOrganization/InviteMember/InviteMember.tsx:44]: done

File: src/pages/ManageOrganization/InviteMember/InviteMember.tsx [hunk 7]
```diff
@@ -90,91 +99,96 @@ export const InviteMember: FC = () => {
...
+                  className={styles.addRowLink}
+                  onClick={() => handleAddNewRow(form)}
+                >
+                  {Messages.addAnotherUser}
+                </Button>
+                <div className={styles.footerContainer}>
+                  <span className={styles.footerMessage}>{Messages.userLimit}</span>
+                  <span>
+                    <Button className={styles.saveButton} onClick={handleModalClose}>
+                      Cancel
```
[the remark this answers is not in the document]
Comment (nicolalamacchia) [reply src/pages/ManageOrganization/InviteMember/InviteMember.tsx:172]: done

**Review by TomasBlazek3PG — APPROVED** [review]

**Pull request merged**